### ISSUE TITLE
Event Emission for Critical State Changes in UpgradeL1Bridge Contract

### ISSUE THREAD

**Issue by NegruGeorge** [tokamak-network/L2-Assets-Migration#20]
**Describe the bug**
In the UpgradeL1Bridge contract, the closer role has the authority to execute several critical functions that can significantly alter the contract's state. However, these functions do not emit events to log these changes, making it difficult to track and audit these state modifications.

Affected Functions:
- function forceRegistry(address[] calldata _position) external onlyCloser
- function forceModify(ForceRegistryParam[] calldata _data) external onlyCloser
- function forceActive(bool _state) external onlyCloser

**Configuration**
- Severity: LOW 

**Impact**
    Events are crucial for tracking state changes and for auditing purposes. Their absence means that any state changes made by the closer role cannot be easily traced, increasing the risk of unnoticed attacks or misconfigurations.
    
**Recommendation**
Emit events in the following functions to log state changes:

- forceActive: Emit an event whenever the active state is toggled.
- forceRegistry: Emit an event for each address registered.
- forceModify: Emit an event for each address whose state is modified.

```
event ActiveStateChanged(bool newState);
event PositionRegistered(address position);
event PositionStateModified(address indexed position, bool newState);
```

Modify the functions to emit these events:

```
function forceActive(bool _state) external onlyCloser {
    active = _state;
    emit ActiveStateChanged(_state); // Emit event
}
```

```
function forceRegistry(address[] calldata _position) external onlyCloser { 
    for (uint i = 0; i < _position.length; i++) {
        position[_position[i]] = true;
        positions.push(_position[i]);
        emit PositionRegistered(_position[i]); // Emit event
    }
}
```

```
function forceModify(ForceRegistryParam[] calldata _data) external onlyCloser {
    for (uint i = 0; i < _data.length; i++) {
        position[_data[i].position] = _data[i].state;
        emit PositionStateModified(_data[i].position, _data[i].state); // Emit event
    }
}
```
**Exploit Scenario **

**Demo**

**Comment by DevUreak** [comment]
Unintentional function calls due to administrator mistakes can cause serious problems. Also, I think a few are needed to implement the front service. We will add it. thank you

**Comment by DevUreak** [comment]
fixed c63f4046c3193d0aedb00428d15afe46de74abe3
